src/lib/types.ts

```diff
@@ -1,20 +1,24 @@
 
-export interface Car { // Car object fields
-    maker: string
-    model: string
-    year: string
-    engine: typeof engineTypes[number]
+export type Car = { // Car object fields
+    maker: typeof carMakers[number] | undefined;
+    model: string | undefined;
+    year: number | undefined;
+    engine: typeof engineTypes[number] | undefined;
+    favorite: boolean;
 }
 
-export interface Motorcycle {
+export type Motorcycle = {
     maker: string;
     model: string;
     year: string;
     engine: string;
+    favorite: boolean;
+
 }
 
 export const engineTypes = ['Petrol', 'Diesel', 'Electric'] // Engine types
+export const carMakers = ['Toyota', 'Honda', 'Ford']
 
 export interface FormErrors extends Car {}
 
-export const emptyCar: Car = { maker: '', model: '', year: '', engine: '' } // Empty car object
\ No newline at end of file
+export const emptyCar: Car = { maker: undefined, model: undefined, year: undefined, engine: undefined, favorite: false } // Empty car object
```

Comment:
Move this constant to the car form files, since it related only to this form. 
No need to declare it in `types` file.